source/common/router/rds_impl.cc

```diff
@@ -174,5 +183,42 @@ Router::RouteConfigProviderSharedPtr RouteConfigProviderManagerImpl::getRouteCon
   return new_provider;
 };
 
+void RouteConfigProviderManagerImpl::addRouteInfo(RdsRouteConfigProviderSharedPtr provider,
+                                                  Buffer::Instance& response) {
+  response.add(fmt::format("route_config_name: {}\n", provider->routeConfigName()));
+  response.add(fmt::format("cluster_name: {}\n", provider->clusterName()));
+  response.add("config dump:\n");
+  response.add(fmt::format("{}\n", provider->configAsString()));
```

Comment:
To what extent do we want to make the results from the various handlers easy to parse? Maybe not for this PR, but it seems to make sense to export out the handlers as JSON, since you don't want to have to scrape manually if writing automation that uses the handlers.